### ISSUE TITLE
loss explotion and become nan

### ISSUE THREAD

**Issue by khoaprogrammer** [experiencor/basic-yolo-keras#16]
Hi, i 'm using full yolo version and try to train the model to predict one class "person" only and i have about 300 images (not load pretrain weight!). i'm using the Adam optimizier with the default value that you use in the code. But when i train the model at 1st epoch, the loss get high very quickly and become nan. Please help!!!

**Comment by experiencor** [comment]
I messed up train_imgs and valid_imgs in train.py. You can try again. Please use the bare python code with caution as it's not quite stable now. The notebook version is recommended at the moment.

**Comment by experiencor** [comment]
@khoaprogrammer You may want to try the latest version. I have well tested on raccoon detection, blood cell detection, and kangaroo detection.
